site/src/pages/WorkspacePage/WorkspaceActions/Buttons.tsx

```diff
@@ -21,19 +20,39 @@ export interface ActionButtonProps {
 	tooltipText?: string;
 }
 
-export const UpdateButton: FC<ActionButtonProps> = ({
+export const UpdateAndStartButton: FC<ActionButtonProps> = ({
 	handleAction,
 	loading,
 }) => {
 	return (
-		<TopbarButton
-			disabled={loading}
-			data-testid="workspace-update-button"
-			onClick={() => handleAction()}
-		>
-			<CloudIcon />
-			{loading ? <>Updating&hellip;</> : <>Update&hellip;</>}
-		</TopbarButton>
+		<Tooltip title="Start workspace with the latest template version.">
+			<TopbarButton
+				disabled={loading}
+				data-testid="workspace-update-button"
+				onClick={() => handleAction()}
+			>
+				<CirclePlayIcon />
+				{loading ? <>Updating&hellip;</> : <>Update&hellip;</>}
+			</TopbarButton>
+		</Tooltip>
+	);
+};
+
+export const UpdateAndRestartButton: FC<ActionButtonProps> = ({
+	handleAction,
+	loading,
+}) => {
+	return (
+		<Tooltip title="Stop workspace, if running, and restart it with the latest template version.">
```

Comment:
```suggestion
		<Tooltip title="Stop workspace and restart it with the latest template version.">
```

more confident wording. unless I'm misunderstanding, that's the point of this change.